csv2ofx/utils.py

```diff
@@ -28,11 +28,11 @@
 ACTION_TYPES = {
     'ShrsIn': ('deposit',),
     'ShrsOut': ('withdraw',),
+    'ReinvDiv': ('reinvest',),
     'Buy': ('buy', 'invest'),
     'Div': ('dividend',),
     'Int': ('interest',),
     'Sell': ('sell',),
-    'ReinvDiv': ('reinvest',),
     'StkSplit': ('split',),
 }
 
```

Comment:
Since dictionaries are unordered, this shouldn't actually make a difference should it??

Reply:
You're right: Python does not guarantee the order of the keys or values.  As it turns out in Python 2.7.14 with the default configuration, it iterated over the keys in the order of the code, and this my fix worked. If you ran this in CPython, you will get it in whatever order it represents the keys.

So the *real* fix would be for me to use an OrderedDict from the python collections module to force a specific order so that the substring matching will hit the larger string first...

Time to update my pull request :)

Reply:
Quick question: I currently made my test a doctest. Would you prefer a scripttest?

Reply:
doctest is fine. did you confirm that current codebase fails that test?

Reply:
Yes (at least on my system, which is Python 2.7.14 under cygwin).  I didn't push the change to see if it fails on Travis' docker images.

Reply:
good enough for me :)